### DISCUSSION TITLE
Dont show a repo under stars if it is added under a List

### DISCUSSION THREAD

**Discussion by santoshyadavdev** [community/community#8291] [Other Feature Feedback, Questions, & Ideas] [answered]
If I have added a repo under a topic showing them again under stars will only cause duplication, for example in the below image the nx-tools repo is already part of my List
![image](https://user-images.githubusercontent.com/11923975/144210317-91987c94-04e3-4b25-b0ab-87f477bb26f4.png)

**Comment by bailey** [comment, marked as answer]
Hi @santoshyadavdev! We've developed Lists to be an added layer on top of the existing starring feature. This means you will still have that overall list you're seeing that contains all starred repositories alongside the lists you create where you can curate a bit further. 

Hope you're enjoying Lists so far! ✨
